developer/src/kmc/src/commands/build.ts

```diff
@@ -79,8 +79,11 @@ async function build(filename: string, options: CompilerOptions): Promise<boolea
       }
     }
 
+    const failureCodes = [CompilerErrorSeverity.Fatal, CompilerErrorSeverity.Error];
+    // TODO: #9100: .concat(options.compilerWarningsAsErrors ? [CompilerErrorSeverity.Warn] : []);
```

Comment:
Just highlighting this, as it doesn't seem to be highlighted in the description... and GitHub will help link this TODO by tagging the issue here: #9100.